lib/libzpool/uzfs_io.c

```diff
@@ -218,3 +253,16 @@ uzfs_flush_data(zvol_state_t *zv)
 {
 	zil_commit(zv->zv_zilog, ZVOL_OBJ);
 }
+
+void
+uzfs_set_rebuilding_mode(zvol_state_t *zv)
```

Comment:
Suggestion@ We can make it little generic by passing enum to set a particular state on zv.

Reply:
ok

Reply:
nice suggestion. I will make it generic.